docker-compose.yaml

```
@@ -18,7 +19,8 @@ services:
       #- ${PWD}/.users.yaml:/config/.users.yaml
     networks:
       - argilla
-
+    volumes:
+      - data:/var/lib/argilla
```

Also, we can add a comment noticing the volume could be modified when the `ARGILLA_HOME_PATH` is modified.
```suggestion
     # If the ARGILLA_HOME_PATH is customized, this value should be change....
      - data:/var/lib/argilla
```

---

Done.